packages/jest-resolve/src/index.js

```diff
@@ -91,8 +91,8 @@ class Resolver {
 
   static findNodeModule(path: Path, options: FindNodeModuleConfig): ?Path {
     const resolver = options.resolver
-      /* $FlowFixMe */
-      ? require(options.resolver)
+      ? /* $FlowFixMe */
```

Comment:
No, it was in a single line before, it was multilined when I made it into a ternary